draft-poidt-ccamp-actn-poi-pluggable.txt

```diff
@@ -0,0 +1,1456 @@
+
+      MDSC is asked to isolate part of the optical network for
+      maintenance and coordinate the O-PNC and P-PNC to preserve the
+      traffic during the maintenance operation.
+
+   In the following sub-sections a workflow for each use case is
+   described.  Each workflow is analyzed consider a scenario like the
+   one described in option 2.  In those cases when the workflow differs
+   between option 1 and option 2, a note describing the differences is
+   added.
+
+
+
+
+
+
+
+
+
+
+
+Galimberti, et al.       Expires 1 December 2023               [Page 13]
+
+Internet-Draft  Applicability of Abstraction and Control        May 2023
+
+
+3.1.  Inter Domain Link discovery and provisioning
+
+   The inter-domain links are set in the Packet and DWDM nodes either
+   manually (e.g. via CLI or NMS) during the installation phase when the
+   operator connects the Pluggable Transceiver to the ROADM port via
+   fiber patch cord or is defined by the MDSC controller and provisioned
+   via the PNCs.  One method and model to define the Inter Layer Link
+   is, for example, to assign a value to the patchcord (for Tx and RX
+   directions) and store those values in the Pluggable and ROADM port
+   provisioning when the fiber is connected between the two ports.  This
+   allows the PNCs to retrieve the values and share them with the MDSC
+   for the correlation and check.  Other smarter and automatic methods
+   of patchcord discovery may be defined but are outside of this draft
+   scope.
+
+   The inter-domain link must be set (or clear) any time a new pluggable
+   module is installed (or removed) and it is connected to the ROADM
+   port with the fiber patchcord.  When a new coherent pluggable
+   interface is installed an inventory notification must be reported to
+   the PNC and MDSC, the reported info are:
+
+           - Pluggable port-ID (e.g. rack/shelf/slot/port or UUID)
+           - Supported Operational-modes
+           - Vendor OUI (if the operational mode in not standard)
+           - Pluggable part number (if the op-mode in not standard)
+           - Manufacturing data
+
+   It would be also possible to auto-discover the inter-domain (inter-
+   domain) links between DWDM coherent pluggables and ROADM ports by
+   checking the input/output power levels (and probably switching on/off
+   the lasers of the pluggables).  This would require the help of MDSC,
+   O-PNC and P-PNC.  The same method could be used to verify the
+   provisioned connectivity.  For further study in this draft.
+
+   In this use case no difference between option 1 and 2 is foreseen,
+   the MDSC can discover the inter-domain links correlating the
+   information received by the P-PNC and O-PN
+
+3.2.  Network topology discovery and provisioning
+
+   The first operation executed by the P-PNC and O-PNC is to discover
+   the network topology and share it with the MDSC via the MPI.  The
+   PNCs will discover and share also the inter-layer links (or
```

Comment:
The section above says:


It would be also possible to auto-discover the inter-domain
   links between DWDM coherent pluggables and ROADM ports by
   checking the input/output power levels (and probably switching on/off
   the lasers of the pluggables).  This would require the help of MDSC,
   O-PNC and P-PNC.  The same method could be used to verify the
   provisioned connectivity. For further study in this draft.

What is missing? Maybe describe better the interaction between MDSC, O-PNC and P-PNC ?